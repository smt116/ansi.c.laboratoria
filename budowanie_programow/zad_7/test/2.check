Podaj liczbe rzeczywista: Pierwiastek z 22.000000 = 4.690416
Liczba odwrotna do 22.000000 = 0.045455
