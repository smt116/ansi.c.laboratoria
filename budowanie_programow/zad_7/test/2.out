Podaj liczbe rzeczywista: Pierwiastek z 0.000000 = 0.000000
Liczba 0.000000 nie ma liczby odwrotnej